README.md

```diff
@@ -19,10 +21,53 @@ UserSimCRS follows the architecture of a typical task-based dialogue system, whi
 
 We refer to the [documentation](https://iai-group.github.io/UserSimCRS/) for details. Specifically, see [this page](https://iai-group.github.io/UserSimCRS/setup_agent.html) on how to set up an existing agent to be evaluated using UserSimCRS.
 
+## Installation
+
+The recommended version of Python is 3.9.  
+The easiest way to install UserSimCRS and all of its dependencies is by using pip:
+```shell
+python -m pip install -r requirements/requirements.txt
+```
+To work on the documentation you also need to install other dependencies:
+```shell
+python -m pip install -r requirements/docs_requirements.txt
+```
+## Usage 
+
+A YAML configuration file is necessary to start the MovieBot, see [default configuration](config/default/config_default.yaml) as example.  
+Run the following command to start the simulation:
+```shell
+python -m usersimcrs.run_simulation -c <path_to_config.yaml>
+```
+### Example
+
+This example shows how to run simulation using the default configuration and the [IAI MovieBot](https://github.com/iai-group/MovieBot).
+
+1. Start IAI MovieBot locally
+
+  * Download the IAI MovieBot [here](https://github.com/iai-group/MovieBot/)
+  * Checkout to the 'separate-flask-server' branch
+  * Follow the IAI MovieBot installation instructions
+  * Start the IAI MovieBot locally: `python -m run_bot -c config/moviebot_config_no_integration.yaml`
+
+Note: you need to update the parameter `moviebot_uri` in the configuration in case MovieBot does not run on the default URI (i.e., `http://127.0.0.1:5001`).
+
+2. Run simulation
+
+```shell
+python -m usersimcrs.run_simulation -c config/default/config_default.yaml
+```
+
```

Comment:
Add a line after this stating what this should produce (e.g., X number of files under data/runs).

Reply:
Done.